app/assets/stylesheets/base/defaults.scss

```diff
@@ -108,7 +108,7 @@ a {
     color: $white;
     display: inline-block;
     text-transform: uppercase;
-    font-family: 'proxima-nova', sans-serif;
+    font-family: "proxima-nova", verdana, sans-serif;
```

Comment:
@tatwell I assigned instances of proxima (body) and futura (headers) font famlies to sass variable as suggested. Makes sense to do them and I think we should switch to variables moving forward.